pandas/core/panel.py

```diff
@@ -42,7 +42,7 @@
     axes_single_arg="{0, 1, 2, 'items', 'major_axis', 'minor_axis'}",
     optional_mapper='', optional_axis='', optional_labels='')
 _shared_doc_kwargs['args_transpose'] = (
-    "three positional arguments: each one of\n{ax_single}".format(
+    "{ax_single}\n\tthree positional arguments from given options".format(
```

Comment:
How does this actually render? This doesn't appear to have a type so does this actually fix PR04?

Reply:
Hmm OK - somewhat surprised by that since I don't know what the type is. Keep in mind that Panel is deprecated so any changes to this module will probably be stripped out in next release anyway. So don't put too much more effort into these if you don't have to